task_1/style.css

```
@@ -1,51 +1,141 @@
+@font-face{
+  font-family: "google sans" ;
+  src: url(./fonts/google-sans-cufonfonts-webfont/ProductSans-Light.woff);
+}
+
+@font-face{
+  font-family: "products san" ;
+  src:
+   url(./fonts/FontsFree-Net-ProductSans-Regular.ttf);
+}
+body{
+  background: #E5E5E5;
+}
 
 .wrapper{
-  margin: 0 auto;
-  max-width: 1440px;
+  margin: 0 auto ; 
+  max-width: 1470px;
 }
 
 .header{
-  border: 1px solid black;
+  align-items: center;
+  display: flex;
   height: 80px;
+  justify-content: space-between;
   width: 100%;
 }
 
+.leftHeaderImg{
+  display: flex;
+}
+
+.menueItem:hover{
+  background-color: red;
+}
+
+.active{
+  background-color: yellow;
+}
+
+.navItems{
```

for the class, ids, image names use kabab case and be consistent 
**please update the names for class and ids** 
```suggestion
.navItems{
```